README-usage.md

```diff
@@ -48,6 +58,14 @@ Add this to ``~/.config/fish/completions/foo-bar.fish``:
 eval (env _MPYL_COMPLETE=fish_source mpyl)
 ```
 
+#### YAML auto completion 
+###### Intellij IDEA or PyCharm
+Go to: `Preferences | Languages & Frameworks | Schemas and DTDs | JSON Schema Mappings`
+- Add new schema
+- Add matching schema file from ``src/mpyl/schema/``
```

Comment:
I think it's better to refer to the published schemas. That way people will always be in sync with the latest released version and don't depend on local file paths.
Maybe you could make a list of the schemas and the file patterns they relate e.g.
`*/deployment/project.yml` -> https://vandebron.github.io/mpyl/schema/project.schema.yml 
`mpyl_config.yml` -> https://vandebron.github.io/mpyl/schema/mpyl_config.schema.yml
`run_properties.yml` -> https://vandebron.github.io/mpyl/schema/run_properties.schema.yml